app/filtering.js

```diff
@@ -693,3 +693,14 @@ module.exports.clearAutofillData = () => {
     ses.autofill.clearAutofillData()
   }
 }
+
+module.exports.setCookie = (cookie) => {
```

Comment:
this method should be moved out of filtering and only called on the default session